py-polars/polars/__init__.py

```diff
@@ -202,6 +203,8 @@ def version() -> str:
     "from_records",
     "from_arrow",
     "from_pandas",
+    # testing
```

Comment:
Is this comment on purpose?

Reply:
Yes, in the definition of `__all__`, which is where this change is, I have listed all the exports per module from where it is imported. So `"testing"` is a module we export, imported from `testing`.